backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java

```diff
@@ -185,6 +185,16 @@ public void draw (CGRect rect) {
 		lastFrameTime = System.nanoTime();
 		framesStart = lastFrameTime;
 
+		// enable OpenGL
+		makeCurrent();
+		// OpenGL glViewport() function expects backbuffer coordinates instead of logical coordinates
+		gl20.glViewport(0, 0, screenBounds.backBufferWidth, screenBounds.backBufferHeight);
```

Comment:
Even if it's probably the same, I would leave the old `gl20.glViewport(IOSGLES20.x, IOSGLES20.y, IOSGLES20.width, IOSGLES20.height);` here (including the "hack" comments) so that's easy to see the only thing we're doing is moving the logic inside the `if (!created)`.

Reply:
Wasn't it moved directly? In the `if (!created)` block it was `gl20.glViewport(0, 0, screenBounds.backBufferWidth, screenBounds.backBufferHeight);` too

Reply:
Oh, sorry, checked it too fast and the comparison view tricked me.